setup.py

```diff
@@ -14,7 +14,9 @@
     author='Incuna',
     author_email='admin@incuna.com',
     url='https://github.com/incuna/incuna-test-utils/',
-    install_requires=[],
+    install_requires=[
+        'Pillow==2.8.1',
+    ],
```

Comment:
Can this be an optional dependency? It should also be a range, not an exact version.